libs/core/include/core/byte_array/const_byte_array.hpp

```diff
@@ -595,3 +584,29 @@ ConstByteArray operator+(char const *a, ConstByteArray const &b);
 
 }  // namespace byte_array
 }  // namespace fetch
+
+namespace std {
```

Comment:
This would have been my preferred option, but at some point, this was introduced to the code and since then we have a had it used in a number of places. It is another PR to solve this and high priority atm.